package.json

```diff
@@ -44,7 +44,8 @@
     "test:coverage": "yarn run test --coverage",
     "test:watch": "yarn run test --watch",
     "test:update": "yarn run test --updateSnapshot",
-    "test": "LC_ALL=en_US.UTF-8 jest",
+    "test": "LC_ALL=en_US.UTF-8 jest ./src/components",
```

Comment:
test should test all .{test, spec}.{tsx, ts}  files no ? helpers/ ....

Reply:
I changed it to test `./src/components ./src/helpers ./src/utils`

Reply:
You can use [testPathIgnorePatterns](https://jestjs.io/docs/en/configuration#testpathignorepatterns-array-string) to only exclude this file
This would be more future proof :)

Reply:
And I don't really understand, we can't run the a11y tests with the unit tests ?

Reply:
Thanks I was searching for such option :)

We don't run with unit testing for the simple reason that our accessibility is not good at all yet and will make test fails. To avoid confusion and understand better CI I split a11y and unit testing.

Also a11y tests can only be run on new stories format (using TSX).